src/App.js

```diff
@@ -29,10 +40,12 @@ class App extends Component {
     return (
       <main className = 'App'>
         <Header currentState={this.state.currentMovie}/>
-        {!this.state.currentMovie &&
+        {this.state.error &&
+         <h1>Sorry no movies found 🤷‍♂️</h1>}
+        {!this.state.currentMovie && !this.state.error &&
```

Comment:
Now that we have the functionality of error feedback being displayed to user we can spend the rest of our weekend refactoring.